crates/shared/src/bad_token/token_owner_finder/blocksout.rs

```diff
@@ -0,0 +1,90 @@
+use anyhow::{bail, Result};
+use ethcontract::H160;
+use reqwest::{Client, Url};
+use serde::Deserialize;
+
+use super::TokenOwnerFinding;
+
+const BASE: &str = "https://blockscout.com/";
+
+pub struct BlockscoutTokenOwnerFinder {
+    client: Client,
+    base: Url,
+}
+
+impl BlockscoutTokenOwnerFinder {
+    pub fn try_for_network(client: Client, network_id: u64) -> Result<Self> {
```

Comment:
nit: `try_for_network()` seems a little odd to me because it doesn't specify what the function `tries`.
Maybe `try_new()` or `try_new_for_network()` instead?

Reply:
Typical Rust naming could also include `try_from_network` or `try_with_network`.